ui/vty-lib/src/Ariadne/UI/Vty/Widget/Logs.hs

```diff
@@ -61,12 +60,12 @@ drawLogsWidget LogsWidgetState{..} = do
         height = rdrCtx ^. B.availHeightL
         drawLogMessage (LogMessage message) =
           V.vertCat $
-          csiToVty attr <$> Text.lines message
+          csiToVty attr <$> lines message
         img =
           (if logsWidgetFollow then V.cropTop height else identity) $
           V.vertCat $
           reverse $
-          fmap drawLogMessage $
+          drawLogMessage <$>
```

Comment:
This one is debatable. I'm thinking it should not be changed, but I'm willing to listen to others' opinion.

Reply:
I like `<$>` version more, but I think `fmap` should be acceptable.